README.md

```diff
@@ -135,6 +135,20 @@ git, I worked on github and gitlab. I tried using git guis but I prefer to work
 
 * Explain event delegation
 * Explain how `this` works in JavaScript
+    This points to an object depending on the context,
+    In what function was used in which we defined it.
```

Comment:
Bazowałem na google translate.
"This wskazuje na jakiś obiekt w zależności od kontekstu,
w jakim została użyta funkcja, w której go zdefiniowaliśmy."

Czyli miejsce deklaracji danego obiektu/funkcji.